### PR TITLE
Dependency typings

### PR DESCRIPTION
**Pull request by Zyie**
##### Description of change

Another request from @bigtimebuddy to add typings for the dependencies that dont have them already.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,14 +1,64 @@
...
+        "removeComments": false,
+        "strict": true,
+        "noImplicitAny": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noImplicitReturns": true,
+        "baseUrl": "./",
+        "paths": {
+            "@pixi.js": ["./bundles/pixi.js"],
+            "@pixi.js-legacy": ["./bundles/pixi.js-legacy"],
```
Comment (bigtimebuddy) [inline tsconfig.json]: `@pixi.js` should be `pixi.js` and `@pixi.js-legacy` should be `pixi.js-legacy.

File: tsconfig.json [hunk 2]
```diff
@@ -1,14 +1,64 @@
...
+            "@pixi/accessibility": ["./packages/accessibility/src/"],
```
Comment (bigtimebuddy) [inline tsconfig.json]: Could you provide some more context about why paths to the source file for each package is necessary?

**Comment by bigtimebuddy** [comment]
Thanks @Zyie for doing this. Great work just a couple of comments.

File: tsconfig.json (hunk 2 above)
Comment (Zyie) [reply tsconfig.json]: Yeah so without the paths typescript doesn't know what these modules are. 

Take this line for example, `import { isMobile } from '@pixi/utils';`. We get an error saying typescript cannot find `@pixi/utils`

By using the paths we map `@pixi/utils` to `../../utils/src/`.

Here's a link to explain how it works [path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping)

File: tsconfig.json (hunk 1 above)
Comment (Zyie) [reply tsconfig.json]: oops, all fixed

File: tsconfig.json (hunk 2 above)
Comment (bigtimebuddy) [reply tsconfig.json]: If I remove these paths, I am still able to build (`npm run build`) fine without them. In terms of letting packages know about other packages exports, we should do this through module typings (like a very permissive `declare module "@pixi/blah";` in each package or in a global typings folder we can remove later once we emit typings). Shouldn't we be using module resolution in TS to resolve the paths to the packages?

Comment (Zyie) [reply tsconfig.json]: If you remove the paths and convert say the utils `index.js` to `index.ts` that is when you get the error.

I'll remove them and keep this pr just for the dependency types

**Pull request merged**